source/_docs/installation/freenas.markdown

```diff
@@ -13,24 +13,24 @@ Enter the Home Assistant jail. If you don't know which name you have given the j
 iocage exec HomeAssistant
 ```
 
-Create the user and group that Home Assistant will run as. The user/group ID of `8123` can be replaced if this is already in use in your environment.
-
-```bash
-pw groupadd -n homeassistant -g 8123
-echo 'homeassistant:8123:8123::::::/bin/csh:' | adduser -f -
-```
-
-Install the necessary Python packages and virtualenv:
+Install the suggested packages and virtualenv:
 
 ```bash
 pkg update
 pkg upgrade
-pkg install -y python37 py37-sqlite3 ca_root_nss
+pkg install -y autoconf bash ca_root_nss gmake pkgconf python37 py37-sqlite3
 python3.7 -m ensurepip
 pip3 install --upgrade pip
 pip3 install --upgrade virtualenv
```

Comment:
So why are we still advertising `virtualenv` instead of the build-in `venv` ?

Reply:
That's a good question. I didn't even think about `venv`.

I think the first time I used a virtualenv, the guide I was following must have used `virtualenv`. Typically, I do prefer "minimum you can install" but never considered `virtualenv` to add any considerable overhead so I never looked at the alternatives. That has to be at least 10 in the past by now. So I guess it's just old habits. "If it ain't broke..." I never had a reason to switch.

If you think this should be changed then maybe we should change it. I really don't know any difference so I'd like to ask is there an advantage to `venv` beside it's just built in? I assume everything would still work the same, correct? 

On the opposite side is there a disadvantage to `virtualenv`

Thanks

Reply:
There is no real disadvantage, especially if you consider our project is Python3 only.

An advantage is that you do not need an additional package (unless it is explicitly packaged by the package manager in a separate package, so that needs checking). Furthermore, `venv` does not copy in the Python binaries into the `venv`.

Reply:
I changed it over to venv.

Thanks for explaining, I changed my install script as well. Shaved some seconds off. :smile: